syntax_and_semantics/literals/range.md

```diff
@@ -1,14 +1,32 @@
 # Range
 
-A [Range](http://crystal-lang.org/api/Range.html) is typically constructed with a range literal:
+A [Range](http://crystal-lang.org/api/Range.html) represents an interval between two values. It is typically constructed with a range literal, consisting of two `..` or three dots:
```

Comment:
When there's `` `..` `` after "two", why isn't there `` `...` `` after "three"? I think these can be omitted. You already see right below what two and three dots look like.
```suggestion
A [Range](http://crystal-lang.org/api/Range.html) represents an interval between two values. It is typically constructed with a range literal, consisting of two or three dots:
```